src/libYARP_init/src/CustomInit.cpp

```diff
@@ -37,6 +38,19 @@ extern "C" void yarpCustomFini()
     yarp::os::LogForwarder::clearInstance();
 }
 
+void yarpClockInit()
```

Comment:
:warning: :-1::-1::-1: This is not right here, it should be in libYARP_OS, not in libYARP_init.

The only reason for having libYARP_init is for static yarp builds, do not add anything there.